app/controllers/submissions_controller.rb

```diff
@@ -1,118 +1,81 @@
-      footer_presenter = FooterPresenter.new(Setting.get)
-      render :new, locals: { submission: submission, footer_presenter: footer_presenter }
+      render :new, locals: {
+        submission: Submission.new,
+        answers: build_form_answers(form_questions),
+        footer_presenter: FooterPresenter.new(Setting.get)
+      }
     else
       render :closed
     end
   end
 
   def thank_you
+    render :thank_you
   end
 
   def create
     submission = Submission.new(submission_params)
 
     if submission.valid?
-      submission_rejector = SubmissionRejector.new.reject_if_any_rules_broken(submission)
+      SubmissionRejector.new.reject_if_any_rules_broken(submission)
       submission.save
+      attributes_collection = answers_attributes(answers_params, submission.id)
+      Answer.create_collection(attributes_collection)
 
       redirect_to submissions_thank_you_url
     else
-      footer_presenter = FooterPresenter.new(Setting.get)
-      render :new, locals: { submission: submission, footer_presenter: footer_presenter }
+      render :new, locals: {
+        submission: submission,
+        answers: build_form_answers(form_questions),
+        footer_presenter: FooterPresenter.new(Setting.get)
+      }
     end
   end
 
-  def destroy
-    submission = Submission.find(params[:id])
-    submission.destroy
+  private
 
-    redirect_to submissions_url, notice: 'Submission was successfully destroyed.'
+  def submission_params
+    params.require(:submission).permit(:full_name, :email, :age, :codecademy_username,
+      :description, :html, :css, :js, :ror, :db, :programming_others, :english,
+      :operating_system, :first_time, :goals, :problems)
   end
 
-  private
-    def submission_params
-      params.require(:submission).permit(:full_name, :email, :age,
-      :codecademy_username, :description, :html, :css, :js, :ror, :db,
-      :programming_others, :english, :operating_system, :first_time, :goals,
-      :problems)
-    end
+  def answers_params
+    params.require(:submission)
+      .permit(answers_attributes: [:value, :question_id])[:answers_attributes]
+  end
 
-    def create_rate_presenters(rates)
-      rates.map { |rate| RatePresenter.new(rate, rate.user) }
+  def answers_attributes(answers_params, submission_id)
+    if answers_params
+      answers_params.values.map { |params| params.merge({ submission_id: submission_id }) }
+    else
+      {}
     end
+  end
 
-    def create_comment_presenters(comments)
-      comments.map { |comment| CommentPresenter.new(comment, comment.user) }
-    end
+  def build_form_answers(questions)
+    questions.map { |q| Answer.new(question: q) }
+  end
 
-    def create_submission_presenters(submissions)
-      submissions.map do |submission|
-        SubmissionPresenter.new(
-          submission,
-          submission.rates,
-          SubmissionRepository.new,
-          current_user
-        )
-      end
-    end
+  def form_questions
+    Question.all
+  end
```

Comment:
I think generally idea of using repositories was to use them as an interface to active record, to hide all active record behind it. I also don't have very strong opinion about this, because I'm not 100% sure about this concept, as it's so easy to forget and use Active Record instead.
Personally I would create it even for one method because of that, but your choice.